ditto/readers/cyme/read.py

```diff
@@ -742,7 +744,9 @@ def check_object_in_line(self, line, obj):
 
         return np.any([x in line for x in self.header_mapping[obj]])
 
-    def parser_helper(self, line, obj_list, attribute_list, mapping, *args):
+    def parser_helper(
+        self, line, obj_list, attribute_list, mapping, *args, **kwargs
+    ):  # now only will work with python 3 due to kwargs
```

Comment:
This will still work in Python2. Why do you say it'll only work in Python3? That said, we should drop Python2 support: https://pythonclock.org/

Reply:
This specific comment has been removed. Compatibility with python 2 will have to be addressed later